docs/guide/using-validator.md

```diff
@@ -194,8 +194,8 @@ $result = (new Validator())->validate($data, $rulesProvider);
 
 #### Using interface method implementation
 
-Providing rules via interface method implementation has priority over PHP attributes. So, in case of both present, the 
-attributes will be ignored without causing exception.
+Providing rules via interface method implementation has priority over PHP attributes. So, in the case of both present,
+the attributes will be ignored without causing an exception.
```

Comment:
```suggestion
Providing rules via interface method implementation has priority over PHP attributes. So, in case both are present,
the attributes will be ignored without causing an exception.
```